docs/permissions.md

```diff
@@ -29,6 +29,11 @@ Additionally, users with the **ProjectCoordinator** role can:
 
 ## Releasing Outputs
 
+!!! note
+    Releasing of outputs to the Jobs site is currently being rolled out.
```

Comment:
Should "Jobs site" be "OpenSAFELY Jobs site"? Not sure, but maybe. I'm not sure if we're consistent throughout the documentation.

Not a big problem though.

Reply:
This is the first use of that term, previously we've used `job server`, I'll stick with that to be consistent for now and make a note to use a new term when we rename it.